prototype/wof.js

```diff
@@ -97,12 +98,21 @@ function insertWofRecord( wof, next ){
       }
     }
 
-    // add 'name:*'
-    for( var attr in wof ){
-      // https://github.com/whosonfirst/whosonfirst-names
-      // names: preferred|colloquial|variant|unknown
-      var match = attr.match(/^name:([a-z]{3})_x_(preferred|colloquial|variant)$/);
-      if( match ){
+    // note: skip all `name:*` fields when we suspect that they were sourced from
+    // machine transliteration via WikiData.
+    // see: https://github.com/whosonfirst-data/whosonfirst-data/issues/799
+    const hasDeadOrObscureLanguages = _.has(wof, 'name:vol_x_preferred');
+    const isLowOrUnknownPopulation = _.get(doc, 'population', 0) < LOW_POPULATION_THRESHOLD;
+    const isLikelyTransliterated = (hasDeadOrObscureLanguages && isLowOrUnknownPopulation);
+    if (!isLikelyTransliterated) {
```

Comment:
It's only really these 4 lines which changed, the rest has just been indented by one tabstop.